src/frontend/src/components/EditUser/index.tsx

```diff
@@ -144,13 +241,45 @@ class EditUser extends React.Component<Props, State> {
                 </div>
               </div>
             </div>
-
             <div className="mb5">
               <button className="bg-blue white bg-animate hover-bg-dark-blue fw6 br2 pv3 ph4 db w-100 tc">
                 Update Account
               </button>
             </div>
 
+            <div role="alert" className="w-100">
+              {this.state.missingName === true ? (
+                <p id="name_input_message" className="bg-washed-red mv4 pa3 br3 fw6">
+                  Name is required.
+                </p>
+              ) : null}
+              {this.state.invalidEmail === true ? (
+                <p id="email_message" className="bg-washed-red mv4 pa3 br3 fw6">
+                  Invalid email address.
+                </p>
+              ) : null}
+              {this.state.missingPassword === true ? (
+                <p id="password_input_message" className="bg-washed-red mv4 pa3 br3 fw6">
+                  Password is required.
+                </p>
+              ) : null}
+              {this.state.invalidPassword === true ? (
+                <p id="password_message" className="bg-washed-red mv4 pa3 br3 fw6">
+                  Passwords must be at least 8 characters.
+                </p>
+              ) : null}
+              {this.state.mismatchPasswords === true ? (
+                <p id="mismatch_message" className="bg-washed-red mv4 pa3 br3 fw6">
+                  Passwords do not match.
+                </p>
+              ) : null}
+              {this.state.invalidResponse === true ? (
+                <p id="no_match_message" className="bg-washed-red mv4 pa3 br3 fw6">
+                  Technical difficulties try again later.
+                </p>
+              ) : null}
+            </div>
+
             <div className="bt b--black-20 pt5 mb5">
```

Comment:
```suggestion
            <div className="visually-hidden bt b--black-20 pt5 mb5">
```

Reply:
Until we implement password-reset via email.

Reply:
Got it.